Cargo.lock

```diff
@@ -1353,7 +1396,7 @@ name = "icu_benchmark_memory"
 version = "0.0.0"
 dependencies = [
  "cargo_metadata",
- "clap",
+ "clap 2.34.0",
```

Comment:
Note: We still have a number of other tools that are requiring Clap 2.34.0; if we want to get rid of Clap 2 from the lockfile, we'll need to migrate all of them.

Reply:
I can do that in a followup I guess